fmt/format.h

```diff
@@ -2432,6 +2433,19 @@ struct ArgArray<N, false/*IsPacked*/> {
   template <typename Formatter, typename T>
   static Arg make(const T &value) { return MakeArg<Formatter>(value); }
 };
+#else
+template <std::size_t N>
+struct ArgArray
+{
```

Comment:
nit: please follow the Google Style: https://google.github.io/styleguide/cppguide.html, in particular 

```c++
struct ArgArray {
```

Reply:
I did my best to adhere to the style guide. I'm not sure if single line functions are allowed, but if so, this should be ok now :-).